cadasta/gui/tools/about/content/license.py

```diff
@@ -0,0 +1,53 @@
+# coding=utf-8
+"""Cadasta license about."""
+
+from cadasta.utilities.i18n import tr
+from cadasta.utilities.resources import get_license_path
+from extras import messaging as m
+from extras.messaging import styles
+
+INFO_STYLE = styles.INFO_STYLE
+
+__author__ = 'Irwan Fathurrahman <irwan@kartoza.com>'
+__date__ = '03/01/17'
+
+
+def license_about():
+    """License about for Cadasta.
+
+    :returns: A message object containing helpful information.
+    :rtype: messaging.message.Message
+    """
+
+    message = m.Message()
+    message.add(heading())
+    message.add(content())
+    return message
+
+
+def heading():
```

Comment:
i'm not sure about this because this class encapsulate what should be rendered
https://github.com/inasafe/inasafe/blob/develop/safe/gui/tools/help/batch_help.py